src/model/window-guardian.ts

```diff
@@ -126,6 +126,10 @@ export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => {
 		return false;
 	};
 
+	const contributionsServiceUrl = process?.env?.SDC_URL
+		? process.env.SDC_URL
+		: CAPI.contributionsServiceUrl;
+
```

Comment:
Could potentially do: 

```ts
proces?.env?.SDC_URL ?? CAPI.contributionsServiceUrl;
```

If we can use nullish coalescing on DCR? Either way - looks good to me!

Reply:
Nice, that works